bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -271,6 +284,7 @@ type NodeRegistrationOptions struct {
 	// with Kubernetes version equal to 1.22 and later.
 	// +kubebuilder:validation:Enum=Always;IfNotPresent;Never
 	// +optional
+	// +kubebuilder:validation:MaxLength=16
```

Comment:
Fields with enum tags do not explicitly need length validations for the purposes of CEL. KAL does not flag enum fields

Reply:
Thx! Yeah I went over all strings here to not have to map 60-70 findings manually. Missed that nuance. Remove the MaxLength here